src/collections/Biographies.ts

```diff
@@ -0,0 +1,71 @@
+import { accessByTenant } from '@/access/byTenant'
+import { filterByTenant } from '@/access/filterByTenant'
+import { tenantField } from '@/fields/tenantField'
+import { Biography } from '@/payload-types'
+import { CollectionConfig, TextField, Validate } from 'payload'
+
+// the types like TextFieldValidation are not parameterized by the collection, so they have `unknown` for siblingData
+const validateName: Validate<string, unknown, Biography, TextField> = (name, { siblingData }) => {
+  if (siblingData.user) {
+    if (name) {
+      return 'A name cannot be provided when a user is linked to this biography.'
+    } else {
+      return true
+    }
+  } else {
+    if (!name) {
+      return 'A name must be provided if no user is linked to this biography.'
+    } else {
+      return true
+    }
+  }
```

Comment:
This logic is kind of confusing. Maybe just an explicit `if/else` here to call out the two scenarios we care about
```suggestion
  if (siblingData.user && name) {
    return 'A name cannot be provided when a user is linked to this biography.'
  }
   else if(!siblingData.user && !name) {
     return 'A name must be provided if no user is linked to this biography.'
  } else {
    return true
  }
```